front/src/component/DeletionOfBookingsByTourist.js

```diff
@@ -1,105 +1,30 @@
 import React from 'react';
 import './Display.css';
 import axios from 'axios'
-import ListOfBooksByTourist from './ListOfBooksByTourist.js'
-import BookBarLight from './BookBarLight.js'
-import Sidebar from './Sidebar'
-import { Redirect } from "react-router-dom"
-import TotalBooks from './TotalBooks.js'
+import {useHistory} from 'react-router-dom';
 
-
-const deleteFormData = id => 
-  axios.delete(`http://localhost:8000/api/bookings/tourist/${id}`)
+const deleteFormData = (idtourist,idevent) => 
+  axios.delete(`http://localhost:8000/api/bookings/tourist/${idtourist}/${idevent}`)
     .then(response => {
       (response.status === 200) && (alert("la réservation est supprimée !"))
     })
 
-function DeleteOfBookingsByEvent({date, time, name, id}){
+function DeletionOfBookingsByTourist({date, time, name, idtourist,idevent}){
+  let history=useHistory()
   return(
   <div id="ListOfVacationers">
     <div id="ProfilVacationer">
-    {name===name.slice(0,20)
-      ?<p className="nameVac">{id}-{name}</p>
-      :<p className="nameVac">{id}-{name.slice(0,20)}...</p>}
+    {name===name.slice(0,25)
+      ?<p className="nameVac">{name}</p>
+      :<p className="nameVac">{name.slice(0,25)}...</p>}
     </div>
     <div className="adresseVac">
       <p>{date.slice(8,10)}/{date.slice(5,7)}/{date.slice(2,4)} {time.slice(0,5)}</p>
     </div>
-    <button id="buttonEye" type="button"></button>
-    <button id="buttonEdit" type="button"></button>
-    <button id="buttonDelete" type="button" onClick={() => deleteFormData(id)} type='submit' value='Supprimer'> </button>  
+    <button id="buttonEye" type="button" onClick={event=> { event.preventDefault(); history.push(`/events/${idevent}`) }} href={`/events/${idevent}`}></button>
+    <button id="buttonDelete" type="button" onClick={() => deleteFormData(idtourist,idevent)}></button>  
   </div>
   )
 }
 
-export default DeleteOfBookingsByEvent;
-
-{/*export default class DeletionOfBookingsByTourist extends React.Component {
```

Comment:
So much dead code... it's a code necropolis !

Reply:
I didn't find dead code :( 
I'm not an archaeologist !